examples/fsspec/README.rst

```diff
@@ -1,37 +1,34 @@
 fsspec
 ======
 
-You can use ``fsspec`` to open NREL h5 resource files hosted on AWS S3 on your local computer. In our internal tests, this is slower than the `HSDS <https://nrel.github.io/rex/misc/examples.hsds.html>`_ and `zarr <https://nrel.github.io/rex/misc/examples.zarr.html>`_ examples, but is much easier to set up. This may be a good option for people outside of NREL trying to access small to medium amounts of NREL .h5 data in applications that are not sensitive to IO performance.
+Filesystem utilities from ``fsspec`` enable users outside of NREL to open h5 resource files hosted on AWS S3 on your local computer. In our internal tests, this is slower than the `HSDS <https://nrel.github.io/rex/misc/examples.hsds.html>`_ and `zarr <https://nrel.github.io/rex/misc/examples.zarr.html>`_ examples, but as of ``rex`` version v0.2.92 it requires zero setup beyond installing ``rex`` and ``fsspec`` as described below. This may be a good option for people outside of NREL trying to access small to medium amounts of NREL .h5 data in applications that are not sensitive to IO performance.
 
 For more info on ``fsspec``, read the docs `here <https://filesystem-spec.readthedocs.io/en/latest/>`_
 
 Extra Requirements
 ------------------
 
-You may need some additional software beyond the rex requirements to run this example:
+You may need some additional software beyond the basic ``rex`` install to run this example:
 
 .. code-block:: bash
 
-    pip install fsspec
+    pip install NREL-rex[s3]
 
 Code Example
 ------------
 
-To open an .h5 file hosted on AWS S3, follow the code example below. Here are some caveats to this approach:
+To open an .h5 file hosted on AWS S3, simply use a path to an S3 resource with any of the ``rex`` file handlers:
 
-- Change ``fp`` to your desired AWS .h5 resource paths.
+- Change ``fp`` to your desired AWS .h5 resource paths (find the s3 paths on `OEDI <https://openei.org/wiki/Main_Page>`_ or with the `AWS CLI <https://aws.amazon.com/cli/>`_).
 - Running this example on a laptop, it takes ~14 seconds to read the meta data, and another ~14 seconds to read the GHI timeseries. This may be faster when running on AWS services in the same region hosting the .h5 file. It is much slower when running on the NREL VPN.
-- The ``s3f`` object works like a local .h5 filepath and can be passed to any of the ``rex`` resource handlers, which will handle all of the data scaling and formatting.
 
 .. code-block:: python
 
     import time
-    import fsspec
     from rex import Resource
 
     fp = "s3://nrel-pds-nsrdb/current/nsrdb_1998.h5"
-    s3f = fsspec.open(fp, mode='rb', anon=True, default_fill_cache=False)
-    res = Resource(s3f.open())
+    res = Resource(fp)
```

Comment:
Nice, this is so much cleaner!

Reply:
right? such a great easy option for external parties.